test/SlowTests/Client/Queries/RavenDB_14541.cs

```diff
@@ -0,0 +1,225 @@
+﻿using System;
+using System.Linq;
+using FastTests;
+using Tests.Infrastructure;
+using Xunit.Abstractions;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Queries;
+using Raven.Client.Documents.Session;
+using Xunit;
+
+
+namespace SlowTests.Client.Queries
+{
+    public class RavenDB_14541 : RavenTestBase
+    {
+        public RavenDB_14541(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [RavenFact(RavenTestCategory.Querying | RavenTestCategory.Indexes)]
+        public void RavenQueryIncludeTest1()
```

Comment:
Please change to a meaningful names for tests

Reply:
Done.